app/bundles/LeadBundle/Entity/Tag.php

```diff
@@ -64,6 +75,18 @@ public function getId()
     }
 
     /**
+     * @param int
+     *
+     * @return Tag
+     */
+    public function setId($id)
```

Comment:
ID is generated value (by `autoincrement` in MySQL), so it does not make sense to have setter?

Reply:
It's because of line https://github.com/mautic/mautic/pull/5204/files#diff-88bdfff1de8a013734dd360b3b4f2763R119

Reply:
Could you mock the entity instead then use `->method('getId')->willReturn(1)`?